packages/automerge-repo-react-hooks/src/useLocalAwareness.ts

```diff
@@ -1,19 +1,23 @@
 import { useEffect } from "react"
 import useStateRef from "react-usestateref"
-import { peerEvents } from "./useRemoteAwareness.js"
+import { peerEvents } from "./useRemoteAwareness"
```

Comment:
separate commit?

Reply:
Nah, I think we can roll in other minor cleanup.